test/LinkedList.test.js

```diff
@@ -73,6 +72,13 @@ contract('LinkedList', async () => {
         assert.equal(returnedString, testString, 'values do not match');
     });
 
+    it('test contains() with item', async () => {
+        const testString = 'thing';
+        await list.appendAtHead(testString);
+        
+        assert(await list.appendAtHead(testString), `list does not contain ${testString}`);
+    });
+
```

Comment:
you're missing tests for https://github.com/brianwchou/solidity-linked-list/issues/2

You can test this by walking through until the end of the list in js and then comparing to the size you've saved.